platform_versioned_docs/version-24.1.0/reports/overview.mdx

```diff
@@ -30,6 +30,12 @@ Create a config file that defines the paths to a selection of output files publi
 Any configuration provided in the interface will override configuration supplied in the pipeline repository.
 :::
 
+### Configure reports for Nextflow CLI runs 
+
+Seqera can manage the reports and log files for pipeline runs launched with Nextflow CLI (`nextflow run <pipeline> -with-tower`) when the run is launched with a workspace primary compute environment. Specify the Seqera workspace from which to run the pipeline by setting the `TOWER_WORKSPACE_ID` environment variable. This workspace must have a primary compute environment set. Reports are listed under the **Reports** tab on the run details page. 
+
+Execution logs for pipelines run with a primary compute environment are available in the **Reports** tab by default. To specify additional report files to be made available, your pipeline repository root folder must include a `tower.yml` file that specifies the files to be included (see below).  
```

Comment:
This is not true either. The logs are in the logs tab and not in the reports. The reports tab will include only the reports.